apps/client/src/components/Policy/Media/index.tsx

```diff
@@ -0,0 +1,183 @@
+'use client';
+
+import { BulletItem } from 'client/components';
+import { useGetWindowWidth } from 'client/hooks';
+
+import * as S from './style';
+
+const 설치대수설치위치및촬영범위 = [
+  '설치대수: 45대',
+  '설치 위치 및 촬영범위: 본교 건물 내ㆍ외부 및 주차장',
+];
+
+const 개인영상정보의확인방법및장소에관한사항 = [
+  '확인방법: 영상정보 관리책임자에게 미리 연락하고 개인영상정보(열람, 존재확인) 청구서로 요청한 후 확인 가능합니다.',
+  '확인장소: 행정실',
+];
+const 영상정보의안전성확보조치 = [
+  '귀하는 개인영상정보에 관하여 열람 또는 존재확인, 삭제를 원하는 경우 언제든지 영상정보처리기기 운영자에게 요구하실 수 있습니다. 단, 귀하가 촬영된 개인영상정보 및 명백히 정보주체의 급박한 생명, 신체, 재산의 이익을 위하여 필요한 개인영상정보에 한정됩니다.',
+  '본 기관은 개인영상정보에 관하여 열람 또는 존재확인, 삭제를 요구한 경우 지체없이 필요한 조치를 하겠습니다.',
+];
+
+const MediaPolicy = () => {
+  const width = useGetWindowWidth();
+  return (
```

Comment:
```suggestion
  const width = useGetWindowWidth();
  
  return (
```

사이에 공백 추가하시면 더 깔끔해지겠네요!

Reply:
수정되었습니다 90ce98496cfdf7fe4ad7b13a4498caf7683bc040